<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="1" failures="0" file="scan_utils_tests.py" name="scan_utils_tests.TestScanUtils-20201130173652" skipped="0" tests="1" time="0.004" timestamp="2020-11-30T17:36:52">
	<testcase classname="scan_utils_tests.TestScanUtils" name="test_scan_one_port_all_closed" time="0.004" timestamp="2020-11-30T17:36:52">
		<error message="name 'socket' is not defined" type="NameError">
<![CDATA[Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "/home/student/Documents/cs3280project3/src/unittest/python/scan_utils_tests.py", line 21, in test_scan_one_port_all_closed
    the_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
NameError: name 'socket' is not defined
]]>		</error>
	</testcase>
</testsuite>
